integration/maptest/map.go

```diff
@@ -219,7 +219,7 @@ func RunLeafHistory(ctx context.Context, t *testing.T, tadmin trillian.TrillianA
 		HashStrategy []trillian.HashStrategy
 		set          [][]*trillian.MapLeaf
 		get          []struct {
-			revision  int64
+			revision  uint64
```

Comment:
Even though the revision number will always be positive, it may be better to avoid a bunch of casting and just use int64 everywhere. The API should do initial bounds checking.